### ISSUE TITLE
Add Yoast SEO to WordCamp sites

### ISSUE THREAD

**Issue by ghost** [WordPress/wordcamp.org#575]
Imported from https://meta.trac.wordpress.org/ticket/4803 
Created by @jono-alderson:

Following discussions in #meta-wordcamp, we'd like to add Yoast SEO (Premium) to WordCamp sites. 

This will provide tooling to help with day-to-day management, and help to address some of the serious SEO challenges the sites face (as well as adhering a bit better to SEO best practice).

For example:
- Redirecting a URL
- Noindex'ing a page/post
- Manually selling a canonical URL value
- Managing meta attributes of a page

This requires some consideration around:
- Installation
- Updates
- Permissions

**Comment by ghost** [comment]
[Comment](https://meta.trac.wordpress.org/ticket/4803#comment:1) by [@nukaga](https://profiles.wordpress.org/nukaga/):

About discussion at #meta-wordcamp.
At the beginning of the discussion, we had an issue that Google search for “WordCamp cityname” does not show the 2019 site homepage. The 2018 website will be displayed at the top of the search results.
It seems to the same issue on many WordCamp websites.
If we can solve this problem without changing the operation of the domain name etc. by installing the plug-in, I think that it is a good way.

**Comment by ghost** [comment]
[Comment](https://meta.trac.wordpress.org/ticket/4803#comment:2) by slackbot:

_This ticket was mentioned in [Slack](https://make.wordpress.org/chat/) in #meta-wordcamp by jonoaldersonwp. [View the logs](https://wordpress.slack.com/archives/meta-wordcamp/p1574342839181400)._

**Comment by ghost** [comment]
[Comment](https://meta.trac.wordpress.org/ticket/4803#comment:3) by slackbot:

_This ticket was mentioned in [Slack](https://make.wordpress.org/chat/) in #community-team by imath. [View the logs](https://wordpress.slack.com/archives/community-team/p1576788962130700)._

**Comment by ghost** [comment]
[Comment](https://meta.trac.wordpress.org/ticket/4803#comment:4) by slackbot:

_This ticket was mentioned in [Slack](https://make.wordpress.org/chat/) in #meta-wordcamp by coreymckrill. [View the logs](https://wordpress.slack.com/archives/meta-wordcamp/p1596736715176200)._

**Comment by ghost** [comment]
[Comment](https://meta.trac.wordpress.org/ticket/4803#comment:5) by @iandunn:

xref https://make.wordpress.org/community/2020/03/03/proposal-for-wordcamp-sites-seo-fixes/

**Comment by jdevalk** [comment]
I want to bring this one back into play. We really need to get this plugin on these sites.

I personally think the biggest reason to do it would be for social / search metadata, including opengraph images, not even the redirection of URLs or other things mentioned above.

**Comment by kcristiano** [comment]
I agree it's important that we add an SEO plugin.   I would be glad to see us use Yoast SEO.   If meta has an alternative, we can consider that, but I'd prefer not to keep the status quo of no SEO plugin.
